apps/src/templates/sectionAssessments/MultipleChoiceByStudentSection.jsx

```diff
@@ -0,0 +1,41 @@
+import React, {Component, PropTypes} from 'react';
+import StudentAssessmentOverviewTable from './StudentAssessmentOverviewTable';
+import { studentAnswerDataPropType, questionStructurePropType } from './assessmentDataShapes';
+import {
+  getMultipleChoiceStructureForCurrentAssessment,
+  getStudentMCResponsesForCurrentAssessment,
+} from './sectionAssessmentsRedux';
+import { connect } from 'react-redux';
+
+class MultipleChoiceByStudentSection extends Component {
+  static propTypes = {
+    multipleChoiceStructure: PropTypes.arrayOf(questionStructurePropType),
+    studentAnswerData: PropTypes.arrayOf(studentAnswerDataPropType),
+  };
+
+  render() {
+    const {multipleChoiceStructure, studentAnswerData} = this.props;
+    return (
+      <div>
+        <h2>Multiple choice answers by student section</h2>
```

Comment:
Localization now, or later?

Reply:
This heading isn't permanent, but a helpful thing during prototyping.